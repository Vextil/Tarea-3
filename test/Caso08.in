ListarSolicitudes
Reinicio
ListarSolicitudes
ListarSolicitudes
CancelarSolicitud
Reinicio
IngresarSolicitudes chrome 95 FIN
IngresarSolicitudes firefox 90 FIN
IngresarSolicitudes chrome 90 FIN
CancelarSolicitud
ListarSolicitudes
Reinicio
IngresarSolicitudes chrome 1 FIN
ListarSolicitudes
Reinicio
IngresarSolicitudes chrome 2 FIN
Reinicio
IngresarSolicitudes chrome 4 FIN
EnAlgunoSolicitudes 4 4 1
CrearLista 1 chrome firefox FIN
PerteneceASet 1 1
EnAlgunoSolicitudes 4 5 1
PerteneceASet 1 1
EnAlgunoSolicitudes 5 4 1
PerteneceASet 1 1
EnAlgunoSolicitudes 7 22 1
PerteneceASet 1 1
EnAmbosSolicitudes 4 4 1
PerteneceASet 1 1
EnAmbosSolicitudes 3 4 1
PerteneceASet 1 1
EnAmbosSolicitudes 4 3 1
PerteneceASet 1 1
EnAmbosSolicitudes 98 17 1
PerteneceASet 1 1
SoloEnUnoSolicitudes 4 4 1
PerteneceASet 1 1
SoloEnUnoSolicitudes 99 4 1
PerteneceASet 1 1
SoloEnUnoSolicitudes 4 100 1
PerteneceASet 1 1
SoloEnUnoSolicitudes 12 100 1
PerteneceASet 1 1
ListarSolicitudes
Reinicio
IngresarSolicitudes chrome 47 chrome 74 FIN
EnAlgunoSolicitudes 47 74 1
CrearLista 1 ie chrome FIN
PerteneceASet 1 1
EnAlgunoSolicitudes 74 47 1
PerteneceASet 1 1
EnAlgunoSolicitudes 74 8 1
PerteneceASet 1 1
EnAlgunoSolicitudes 8 74 1
PerteneceASet 1 1
EnAlgunoSolicitudes 14 14 1
PerteneceASet 1 1
EnAmbosSolicitudes 47 74 1
PerteneceASet 1 1
EnAmbosSolicitudes 47 24 1
PerteneceASet 1 1
EnAmbosSolicitudes 24 47 1
PerteneceASet 1 1
EnAmbosSolicitudes 98 17 1
PerteneceASet 1 1
SoloEnUnoSolicitudes 47 74 1
PerteneceASet 1 1
SoloEnUnoSolicitudes 47 88 1
PerteneceASet 1 1
SoloEnUnoSolicitudes 88 47 1
PerteneceASet 1 1
SoloEnUnoSolicitudes 27 10 1
PerteneceASet 1 1
ListarSolicitudes
Reinicio
IngresarSolicitudes chrome 43 slack 3 FIN
ListarSolicitudes
Reinicio
IngresarSolicitudes chrome 75 slack 44 FIN
Reinicio
IngresarSolicitudes chrome 28 slack 49 FIN
EnAlgunoSolicitudes 28 49 1
CrearLista 1 chrome slack dropbox FIN
PerteneceASet 1 1
EnAlgunoSolicitudes 49 15 1
PerteneceASet 1 1
EnAlgunoSolicitudes 15 49 1
PerteneceASet 1 1
EnAlgunoSolicitudes 7 22 1
PerteneceASet 1 1
EnAmbosSolicitudes 28 49 1
PerteneceASet 1 1
EnAmbosSolicitudes 28 28 1
PerteneceASet 1 1
EnAmbosSolicitudes 49 28 1
PerteneceASet 1 1
EnAmbosSolicitudes 98 17 1
PerteneceASet 1 1
SoloEnUnoSolicitudes 28 49 1
PerteneceASet 1 1
SoloEnUnoSolicitudes 49 28 1
PerteneceASet 1 1
SoloEnUnoSolicitudes 28 28 1
PerteneceASet 1 1
SoloEnUnoSolicitudes 49 49 1
PerteneceASet 1 1
SoloEnUnoSolicitudes 28 9 1
PerteneceASet 1 1
SoloEnUnoSolicitudes 9 28 1
PerteneceASet 1 1
SoloEnUnoSolicitudes 49 70 1
PerteneceASet 1 1
SoloEnUnoSolicitudes 70 49 1
PerteneceASet 1 1
SoloEnUnoSolicitudes 5 4 1
PerteneceASet 1 1
ListarSolicitudes
Reinicio
IngresarSolicitudes sublime 30 slack 30 FIN
EnAlgunoSolicitudes 30 30 1
CrearLista 1 thunderbird slack sublime FIN
PerteneceASet 1 1
EnAlgunoSolicitudes 67 30 1
PerteneceASet 1 1
EnAlgunoSolicitudes 30 67 1
PerteneceASet 1 1
EnAlgunoSolicitudes 5 22 1
PerteneceASet 1 1
EnAmbosSolicitudes 30 30 1
PerteneceASet 1 1
EnAmbosSolicitudes 7 30 1
PerteneceASet 1 1
EnAmbosSolicitudes 30 7 1
PerteneceASet 1 1
EnAmbosSolicitudes 98 17 1
PerteneceASet 1 1
SoloEnUnoSolicitudes 30 30 1
PerteneceASet 1 1
SoloEnUnoSolicitudes 4 30 1
PerteneceASet 1 1
SoloEnUnoSolicitudes 30 4 1
PerteneceASet 1 1
SoloEnUnoSolicitudes 28 2 1
PerteneceASet 1 1
ListarSolicitudes
Reinicio
IngresarSolicitudes root 1 ps 100 root 100 ps 1 FIN
EnAlgunoSolicitudes 30 30 1
CrearLista 1 gcc grep root sshd httpd ps FIN
PerteneceASet 1 1
EnAlgunoSolicitudes 100 1 1
PerteneceASet 1 1
EnAlgunoSolicitudes 1 100 1
PerteneceASet 1 1
EnAlgunoSolicitudes 2 1 1
PerteneceASet 1 1
EnAlgunoSolicitudes 2 3 1
PerteneceASet 1 1
EnAmbosSolicitudes 100 1 1
PerteneceASet 1 1
EnAmbosSolicitudes 1 100 1
PerteneceASet 1 1
EnAmbosSolicitudes 100 7 1
PerteneceASet 1 1
EnAmbosSolicitudes 98 17 1
PerteneceASet 1 1
SoloEnUnoSolicitudes 100 1 1
PerteneceASet 1 1
SoloEnUnoSolicitudes 1 100 1
PerteneceASet 1 1
SoloEnUnoSolicitudes 30 100 1
PerteneceASet 1 1
SoloEnUnoSolicitudes 1 2 1
PerteneceASet 1 1
SoloEnUnoSolicitudes 81 38 1
PerteneceASet 1 1
ListarSolicitudes
ListarSolicitudes
Reinicio
IngresarSolicitudes wc 22 ls 100 diff 46 cmp 45 pwd 22 pwd 88 git 23 ls 1 cmp 23 cmp 1 pwd 100 FIN
EnAlgunoSolicitudes 22 23 1
CrearLista 1 ls svn gcc grep git wc diff cmp pwd httpd wc FIN
PerteneceASet 1 1
EnAlgunoSolicitudes 22 22 1
PerteneceASet 1 1
EnAlgunoSolicitudes 23 22 1
PerteneceASet 1 1
EnAlgunoSolicitudes 45 46 1
PerteneceASet 1 1
EnAlgunoSolicitudes 100 22 1
PerteneceASet 1 1
EnAlgunoSolicitudes 98 99 1
PerteneceASet 1 1
EnAmbosSolicitudes 22 88 1
PerteneceASet 1 1
EnAmbosSolicitudes 88 22 1
PerteneceASet 1 1
EnAmbosSolicitudes 100 88 1
PerteneceASet 1 1
EnAmbosSolicitudes 88 100 1
PerteneceASet 1 1
EnAmbosSolicitudes 22 100 1
PerteneceASet 1 1
SoloEnUnoSolicitudes 1 23 1
PerteneceASet 1 1
SoloEnUnoSolicitudes 46 22 1
PerteneceASet 1 1
SoloEnUnoSolicitudes 12 47 1
PerteneceASet 1 1
SoloEnUnoSolicitudes 22 1 1
PerteneceASet 1 1
SoloEnUnoSolicitudes 81 38 1
PerteneceASet 1 1
ListarSolicitudes
ListarSolicitudes
Reinicio
IngresarSolicitudes wc 22 ls 100 diff 46 cmp 45 pwd 22 pwd 88 git 23 ls 1 cmp 23 cmp 1 pwd 100 who 22 time 100 top 83 killall 82 kill 82 cal 23 writer 1 writer 4 writer 2 mysqld 22 python 45 ssh 88 tar 67 gzip 43 top 32 gcc 96 mysqldump 84 scp 22 time 27 who 3 FIN
EnAlgunoSolicitudes 22 23 1
CrearLista 1 ls svn scp time root gcc killall top writer mysql mysqld kill grep who scp ssh git wc mysqldump diff python cmp pwd httpd svn wc cal FIN
PerteneceASet 1 1
EnAlgunoSolicitudes 22 22 1
PerteneceASet 1 1
EnAlgunoSolicitudes 100 22 1
PerteneceASet 1 1
EnAlgunoSolicitudes 45 43 1
PerteneceASet 1 1
EnAlgunoSolicitudes 1 2 1
PerteneceASet 1 1
EnAlgunoSolicitudes 37 94 1
PerteneceASet 1 1
EnAmbosSolicitudes 66 77 1
PerteneceASet 1 1
EnAmbosSolicitudes 1 4 1
PerteneceASet 1 1
EnAmbosSolicitudes 32 83 1
PerteneceASet 1 1
EnAmbosSolicitudes 88 83 1
PerteneceASet 1 1
EnAmbosSolicitudes 22 27 1
PerteneceASet 1 1
SoloEnUnoSolicitudes 22 27 1
PerteneceASet 1 1
SoloEnUnoSolicitudes 1 4 1
PerteneceASet 1 1
SoloEnUnoSolicitudes 82 82 1
PerteneceASet 1 1
SoloEnUnoSolicitudes 45 22 1
PerteneceASet 1 1
SoloEnUnoSolicitudes 23 45 1
PerteneceASet 1 1
ListarSolicitudes
ListarSolicitudes
Reinicio
Fin
