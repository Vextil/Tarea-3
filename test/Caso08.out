1>No hay solicitudes
2>Reiniciando
3>No hay solicitudes
4>No hay solicitudes
5> 0
6>Reiniciando
7>OK
8>OK
9>OK
10> 90
11>(90,chrome) (95,chrome) 
12>Reiniciando
13>OK
14>(1,chrome) 
15>Reiniciando
16>OK
17>Reiniciando
18>OK
19>OK
20>OK
21>SI NO 
22>OK
23>SI NO 
24>OK
25>SI NO 
26>OK
27>NO NO 
28>OK
29>SI NO 
30>OK
31>NO NO 
32>OK
33>NO NO 
34>OK
35>NO NO 
36>OK
37>NO NO 
38>OK
39>SI NO 
40>OK
41>SI NO 
42>OK
43>NO NO 
44>(4,chrome) 
45>Reiniciando
46>OK
47>OK
48>OK
49>NO SI 
50>OK
51>NO SI 
52>OK
53>NO SI 
54>OK
55>NO SI 
56>OK
57>NO NO 
58>OK
59>NO SI 
60>OK
61>NO NO 
62>OK
63>NO NO 
64>OK
65>NO NO 
66>OK
67>NO NO 
68>OK
69>NO SI 
70>OK
71>NO SI 
72>OK
73>NO NO 
74>(47,chrome) (74,chrome) 
75>Reiniciando
76>OK
77>(3,slack) (43,chrome) 
78>Reiniciando
79>OK
80>Reiniciando
81>OK
82>OK
83>OK
84>SI SI NO 
85>OK
86>NO SI NO 
87>OK
88>NO SI NO 
89>OK
90>NO NO NO 
91>OK
92>NO NO NO 
93>OK
94>SI NO NO 
95>OK
96>NO NO NO 
97>OK
98>NO NO NO 
99>OK
100>SI SI NO 
101>OK
102>SI SI NO 
103>OK
104>NO NO NO 
105>OK
106>NO NO NO 
107>OK
108>SI NO NO 
109>OK
110>SI NO NO 
111>OK
112>NO SI NO 
113>OK
114>NO SI NO 
115>OK
116>NO NO NO 
117>(28,chrome) (49,slack) 
118>Reiniciando
119>OK
120>OK
121>OK
122>NO SI SI 
123>OK
124>NO SI SI 
125>OK
126>NO SI SI 
127>OK
128>NO NO NO 
129>OK
130>NO SI SI 
131>OK
132>NO NO NO 
133>OK
134>NO NO NO 
135>OK
136>NO NO NO 
137>OK
138>NO NO NO 
139>OK
140>NO SI SI 
141>OK
142>NO SI SI 
143>OK
144>NO NO NO 
145>(30,sublime) (30,slack) 
146>Reiniciando
147>OK
148>OK
149>OK
150>NO NO NO NO NO NO 
151>OK
152>NO NO SI NO NO SI 
153>OK
154>NO NO SI NO NO SI 
155>OK
156>NO NO SI NO NO SI 
157>OK
158>NO NO NO NO NO NO 
159>OK
160>NO NO SI NO NO SI 
161>OK
162>NO NO SI NO NO SI 
163>OK
164>NO NO NO NO NO NO 
165>OK
166>NO NO NO NO NO NO 
167>OK
168>NO NO NO NO NO NO 
169>OK
170>NO NO NO NO NO NO 
171>OK
172>NO NO SI NO NO SI 
173>OK
174>NO NO SI NO NO SI 
175>OK
176>NO NO NO NO NO NO 
177>(1,root) (1,ps) (100,ps) (100,root) 
178>No hay solicitudes
179>Reiniciando
180>OK
181>OK
182>OK
183>NO NO NO NO SI SI NO SI SI NO SI 
184>OK
185>NO NO NO NO NO SI NO NO SI NO SI 
186>OK
187>NO NO NO NO SI SI NO SI SI NO SI 
188>OK
189>NO NO NO NO NO NO SI SI NO NO NO 
190>OK
191>SI NO NO NO NO SI NO NO SI NO SI 
192>OK
193>NO NO NO NO NO NO NO NO NO NO NO 
194>OK
195>NO NO NO NO NO NO NO NO SI NO NO 
196>OK
197>NO NO NO NO NO NO NO NO SI NO NO 
198>OK
199>NO NO NO NO NO NO NO NO SI NO NO 
200>OK
201>NO NO NO NO NO NO NO NO SI NO NO 
202>OK
203>NO NO NO NO NO NO NO NO SI NO NO 
204>OK
205>SI NO NO NO SI NO NO NO NO NO NO 
206>OK
207>NO NO NO NO NO SI SI NO SI NO SI 
208>OK
209>NO NO NO NO NO NO NO NO NO NO NO 
210>OK
211>SI NO NO NO NO SI NO SI SI NO SI 
212>OK
213>NO NO NO NO NO NO NO NO NO NO NO 
214>(1,ls) (1,cmp) (22,wc) (22,pwd) (23,git) (23,cmp) (45,cmp) (46,diff) (88,pwd) (100,ls) (100,pwd) 
215>No hay solicitudes
216>Reiniciando
217>OK
218>OK
219>OK
220>NO NO SI NO NO NO NO NO NO NO SI NO NO SI SI NO SI SI NO NO NO SI SI NO NO SI SI 
221>OK
222>NO NO SI NO NO NO NO NO NO NO SI NO NO SI SI NO NO SI NO NO NO NO SI NO NO SI NO 
223>OK
224>SI NO SI SI NO NO NO NO NO NO SI NO NO SI SI NO NO SI NO NO NO NO SI NO NO SI NO 
225>OK
226>NO NO NO NO NO NO NO NO NO NO NO NO NO NO NO NO NO NO NO NO SI SI NO NO NO NO NO 
227>OK
228>SI NO NO NO NO NO NO NO SI NO NO NO NO NO NO NO NO NO NO NO NO SI NO NO NO NO NO 
229>OK
230>NO NO NO NO NO NO NO NO NO NO NO NO NO NO NO NO NO NO NO NO NO NO NO NO NO NO NO 
231>OK
232>NO NO NO NO NO NO NO NO NO NO NO NO NO NO NO NO NO NO NO NO NO NO NO NO NO NO NO 
233>OK
234>NO NO NO NO NO NO NO NO SI NO NO NO NO NO NO NO NO NO NO NO NO NO NO NO NO NO NO 
235>OK
236>NO NO NO NO NO NO NO SI NO NO NO NO NO NO NO NO NO NO NO NO NO NO NO NO NO NO NO 
237>OK
238>NO NO NO NO NO NO NO NO NO NO NO NO NO NO NO NO NO NO NO NO NO NO NO NO NO NO NO 
239>OK
240>NO NO NO NO NO NO NO NO NO NO NO NO NO NO NO NO NO NO NO NO NO NO NO NO NO NO NO 
241>OK
242>NO NO SI SI NO NO NO NO NO NO SI NO NO SI SI NO NO SI NO NO NO NO SI NO NO SI NO 
243>OK
244>SI NO NO NO NO NO NO NO NO NO NO NO NO NO NO NO NO NO NO NO NO SI NO NO NO NO NO 
245>OK
246>NO NO NO NO NO NO NO NO NO NO NO NO NO NO NO NO NO NO NO NO NO NO NO NO NO NO NO 
247>OK
248>NO NO SI NO NO NO NO NO NO NO SI NO NO SI SI NO NO SI NO NO SI SI SI NO NO SI NO 
249>OK
250>NO NO NO NO NO NO NO NO NO NO NO NO NO NO NO NO SI NO NO NO SI NO NO NO NO NO SI 
251>(1,ls) (1,cmp) (1,writer) (2,writer) (3,who) (4,writer) (22,wc) (22,pwd) (22,who) (22,mysqld) (22,scp) (23,git) (23,cmp) (23,cal) (27,time) (32,top) (43,gzip) (45,cmp) (45,python) (46,diff) (67,tar) (82,killall) (82,kill) (83,top) (84,mysqldump) (88,pwd) (88,ssh) (96,gcc) (100,ls) (100,pwd) (100,time) 
252>No hay solicitudes
253>Reiniciando
254>Fin
